cpp/include/cudf/strings/char_types/char_types.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+namespace strings
+{
+
+/**
+ * @brief Character type values.
+ * These types can be or'd to check for combination of types
+ */
+enum string_character_types {
+    DECIMAL  =  1,
+    NUMERIC  =  2,
+    DIGIT    =  4,
+    ALPHA    =  8,
+    ALPHANUM = 15,
+    SPACE    = 16,
+    UPPER    = 32,
+    LOWER    = 64
+};
+
+/**
+ * @brief Returns a boolean column containing identifying strings entry in which all
+ * the characters for that string are of the type specified. The output row entry
```

Comment:
Blank line after single-sentence brief. (keep @brief brief).

```suggestion
 *
 * The output row entry
```